pep-0654.rst

```diff
@@ -3,7 +3,7 @@ Title: Exception Groups and except*
 Author: Irit Katriel <iritkatriel@gmail.com>,
         Yury Selivanov <yury@edgedb.com>,
         Guido van Rossum <guido@python.org>
-Status: Accepted
+Status: Final
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 22-Feb-2021
```

Comment:
Additionally, the canonical-doc directive should be added right below the headers to point to the canonical documentation (the reference and the tutorial):

```rst
.. canonical-doc:: :ref:`python:lib-exception-groups` and :ref:`python:except_star`

   See :ref:`python:tut-exception-groups` for a user-focused tutorial.
```

Also, since this PR depends upon the CPython PR python/cpython#106465 anyway, I added the proper section link instead of the link to the individual class.